bokeh/embed.py

```diff
@@ -286,8 +286,8 @@ def file_html(models,
     Args:
         models (Model or Document or list) : Bokeh object or objects to render
             typically a Model or Document
-        resources (Resources) : a resource configuration for Bokeh JS & CSS assets. Pass ``None`` if
-            using js_resources of css_resources manually.
+        resources (Resources) : a resource configuration for Bokeh JS & CSS assets. To use
```

Comment:
Or maybe the type should say it all, i.e. `Resources or tuple(JSResources or None, CSSResources or None)`.